packages/webdriverio/src/types.ts

```diff
@@ -0,0 +1,35 @@
+export type ElementReferenceId = 'element-6066-11e4-a52e-4f735466cecf'
+
+export type ElementReference = Record<ElementReferenceId, string>
+
+export interface ElementObject extends ElementReference, WebdriverIO.BrowserObject {
+    /**
+     * WebDriver element reference
+     */
+    elementId: string
```

Comment:
wasn't one of these the JSON Wire reference?

Reply:
`ELEMENT` is, yeah!